src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ComposeStore.targets

```diff
@@ -421,6 +421,7 @@ Copyright (c) .NET Foundation. All rights reserved.
                           ResolvedFiles ="@(_ResolvedCopyLocalPublishAssets)"
                           PackagesToPrune="$(PackagesToPrune)"
                           TargetFramework="$(TargetFrameworkMoniker)"
```

Comment:
I think you want `TargetFramework` here. 

Then you can ask NuGet about the framework based on the alias.